docs/src/components/CopyMarkdown.tsx

```
@@ -0,0 +1,250 @@
+        case 'em':
+        case 'i':
+          return `*${children}*`;
+        case 'code':
+          // Check if it's inline code (not in a pre block)
+          if (el.parentElement?.tagName.toLowerCase() !== 'pre') {
+            return `\`${children}\``;
+          }
+          return children;
+        case 'pre':
+          // Try to get the language from class
+          const codeEl = el.querySelector('code');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)